airflow/utils/context.py

```diff
@@ -88,6 +88,10 @@
     "var",
     "yesterday_ds",
     "yesterday_ds_nodash",
+    "task_list",
+    "blocking_task_list",
+    "slas",
+    "blocking_tis",
```

Comment:
I don't like that we have this in the context. We shouldn't.

Reply:
> Can we just create a dict and pass to self.render_template_fields(_context)?
Yeah

Reply:
Made the change please review.